sf/cosmos/transform/v1/transform.proto

```diff
@@ -11,3 +11,7 @@ message EventTypeFilter {
 message MessageTypeFilter {
   repeated string message_types = 1;
 }
+
+message EventOriginFilter {
+  repeated string event_origin = 1;
```

Comment:
If we support passing multiple origins here, the name of the field should be `event_origins`.

Reply:
good shout. i'll quickly fix it and remerge

Reply:
https://github.com/figment-networks/proto-cosmos/pull/6